.github/scripts/docker-compose.yml

```diff
@@ -21,3 +21,9 @@ services:
       test: "mysql -uroot -proot ghost -e 'select 1'"
       interval: 1s
       retries: 120
+  redis:
+    image: redis:7.2
```

Comment:
Would you be able to use `7.0`? It's what we currently use elsewhere, so it makes sense to keep those aligned for now

Reply:
👍 updated in https://github.com/TryGhost/Ghost/pull/20085/commits/7b01bb8fe4f8a59ce54d8fca51fb49340a34c589